cypress/e2e/models/migration/controls/stakeholdergroups.ts

```diff
@@ -21,6 +21,7 @@ import {
     deleteAction,
     editAction,
     migration,
+    SEC,
```

Comment:
It looks like SEC is not being used anywhere in file .

Reply:
Removed